tests/test_meshing_queries.py

```diff
@@ -0,0 +1,123 @@
+from ansys.fluent.core import examples
+
+import_filename = examples.download_file(
+    "elbow.cas.h5", "pyfluent/examples/DOE-ML-Mixing-Elbow"
+)
+
+
+def test_meshing_queries(new_mesh_session):
+    meshing_session = new_mesh_session
+    meshing_session.tui.file.read_case(import_filename)
+
+    assert meshing_session.meshing_queries.GetFaceZoneAtLocation(
+        [1.4, 1.4, 1.4]
+    ) == meshing_session.scheme_eval.scheme_eval(
+        "(get-face-zone-at-location '(1.4 1.4 1.4))"
+    )
+
+    assert meshing_session.meshing_queries.GetZonesOfType(
+        "velocity-inlet"
+    ) == meshing_session.scheme_eval.scheme_eval("(get-zones-of-type 'velocity-inlet)")
+
+    assert meshing_session.meshing_queries.GetZonesOfGroup(
+        "inlet"
+    ) == meshing_session.scheme_eval.scheme_eval("(get-zones-of-group 'inlet)")
+
+    assert meshing_session.meshing_queries.GetFaceZonesOfFilter(
+        "*"
+    ) == meshing_session.scheme_eval.scheme_eval("(get-face-zones-of-filter '*)")
+
+    assert meshing_session.meshing_queries.GetCellZonesOfFilter("*") == [3460]
+    assert meshing_session.scheme_eval.scheme_eval("(get-cell-zones-of-filter '*)") == (
+        3460,
+    )
+
+    assert meshing_session.meshing_queries.GetEdgeZonesOfFilter("*") == []
+    assert (
+        meshing_session.scheme_eval.scheme_eval("(get-edge-zones-of-filter '*)") == None
+    )
+
+    assert meshing_session.meshing_queries.GetNodeZonesOfFilter(
+        "*"
+    ) == meshing_session.scheme_eval.scheme_eval("(get-node-zones-of-filter '*)")
+
+    assert meshing_session.meshing_queries.GetObjectsOfType("mesh") == ["elbow-fluid"]
+    # assert meshing_session.scheme_eval.scheme_eval("(get-objects-of-type 'mesh)") == (elbow-fluid,)
```

Comment:
Here scheme returns (elbow-fluid,) hence it is not comparable directly.

Reply:
@hpohekar Is `(elbow-fluid,)` a list of symbols? We can write some utility function to convert list/tuple to set and compare the set. If the individual members are symbols (of type `scheme_eval.Symbol`) we can replace with their `Symbol.str` attribute.

Reply:
I see. Yes, it will save time for remaining rpcs.

Reply:
I was not aware about use of `scheme_eval.Symbol`. Thank you.

Reply:
In fact, we do not *have to* incorporate Scheme code into these tests - bear in mind that we are really testing an API independently of its implementation. It's acceptable to get all the expected results up front and assert them in the test hard-coded.

Reply:
Done. Thank you very much. It will save lot of time.